cli/responseemitter.go

```diff
@@ -168,29 +210,15 @@ func (re *responseEmitter) Emit(v interface{}) error {
 		return io.ErrClosedPipe
 	}
 
-	if err, ok := v.(cmdkit.Error); ok {
-		v = &err
-	}
-
 	var err error
 
 	switch t := v.(type) {
 	case *cmdkit.Error:
-		_, err = fmt.Fprintln(re.stderr, "Error:", t.Message)
-		if t.Code == cmdkit.ErrFatal {
-			defer re.Close()
-		}
-		re.wLock.Lock()
-		defer re.wLock.Unlock()
-		re.exit = 1
-
-		return nil
-
+		log.Errorf("got an error: %v\n%s", err, debug.Stack())
```

Comment:
Again, we should probably forward these errors to `CloseWithError` for now.

Reply:
I believe we should do away with emitting errors and make sure doing so is an error.

Reply:
We should. However, we should either *panic* in cases like this or "do the right thing". Simply logging and dropping the error on the floor is not going to end well.